packages/deployer/src/server/handlers/__tests__/mcp.test.ts

```diff
@@ -0,0 +1,151 @@
+// packages/deployer/src/server/handlers/__tests__/mcp.test.ts
+
```

Comment:
```suggestion
```

This test is good, but we should also have an itest in `packages/mcp/src/server.test.ts` that starts Mastra w/ an MCPServer attached, and then an MCPClient connects to it and is able to call a tool and get a reponse